docs/advanced.md

```diff
@@ -447,6 +447,20 @@ yargs.parserConfiguration({
 See the [yargs-parser](https://github.com/yargs/yargs-parser#configuration) module
 for detailed documentation of this feature.
 
+## Command finish hook
+### Example
+```
+yargs
+    .comand('cmd', ..., async () => {
+        await this.model.find()
+        return Promise.resolve('result value')
+    })
+    .onFinishCommand(async (resultValue) => {
```

Comment:
Instead of a general yargs.onFinishCommand(), wouldn't we cover more use cases by adding an optional parameter to yargs.command(), which would be the finishCommand handler for this command?

EDIT: After a 2nd thought, a per-command handler does not make sense, as it is already easy, when writing an async command handler, to handle the termination in the handler itself.

And a per-command handler does not cover the commandDir use case, when the main yargs file does not have the hand on how the command handlers are written. Your general yargs.onFinishCommand does.

So keep it as is.